Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "qemu_dxe_core"
-version = "2.0.4"
+version = "2.1.0"

```

Comment:
Should this be changed? I assumed this would be changed when we make a release in that repo